services/nginz/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN cd /src/libzauth/libzauth-c \
     && make install
 
 # Nginz container
-FROM alpine:3.11
+FROM alpine:3.10
```

Comment:
What does "not supported" mean? Perhaps leave a comment for the next person who feels like bumping this number? What went wrong?